v6_MIGRATION_GUIDE.md

```diff
@@ -244,3 +245,8 @@ Kotlin:
         // handle result
     }
 ```
+
+## Browser Switch
+
+DropIn handles browser switching internally. Any payment method that requires a browser switch will work automatically.
```

Comment:
Do they no longer need to include an intent-filter in their AndroidManifest.xml? If so, I think we should call that out specifically so they know what to remove.

Reply:
Yeah good call, i'll update. This slipped through the cracks I don't think an intent filter was ever needed 😅 .

Reply:
Definitely yeah thanks for finding this. We did it for 3DS but I missed that one, I'll update.